adguard.json

```diff
@@ -0,0 +1,82 @@
+{
+    "Adguard-home": {
+        "containers": {
+            "adguard": {
+                "image": "adguard/adguardhome",
+                "launch_order": 1,
+                "opts": [
+                    [
+                        "--net=adguard-home",
```

Comment:
I think a change is needed to:
 "--net",
"adguard-home"

Reply:
@kanecko at least in the embyserver.json & the ESPHome.json it is done the same way. In the home-assistant-linuxserver it is done as you suggested. So I guess the only reason to change it would be to make it the same in all rock-ons. Or if you want to keep a defined format.

Reply:
If I remember correctly either syntax is valid as in the end the `docker run ...` syntax itself is lenient with it (`--net host`, `--net=host`, `--network host`, etc...).